src/ci/docker/run.sh

```diff
@@ -269,7 +269,7 @@ touch $objdir/${SUMMARY_FILE}
 
 extra_env=""
 if [ "$ENABLE_GCC_CODEGEN" = "1" ]; then
-  extra_env="$extra_env --env ENABLE_GCC_CODEGEN=1"
+  extra_env="$EXTRA_ENV --env ENABLE_GCC_CODEGEN=1"
```

Comment:
I don't understand why you put this back -- Is there an `$EXTRA_ENV` coming from somewhere else?

Reply:
I was just confirming that it was not that. The problem currently is that we use the old mangling version somewhere and I'm trying to understand where. I'll remove/replace this commit often I think. Sorry for the noise. ^^'